ui/src/cpu_inspector.rs

```diff
@@ -58,10 +58,10 @@ impl<T: Cpu> View for CpuInspector<T> {
         ui.heading("Registers");
         ui.add_space(8.0);
 
-        let registers = match self.gba.lock() {
-            Ok(gba) => gba.cpu.registers(),
-            Err(_) => vec![],
-        };
+        let registers = self
```

Comment:
Why this change? No problem about it, just for me it is more readable the first version. I don't know if there is some kind improvement.

Reply:
More idiomatic, and also clippy complains :D